_includes/head.html

```diff
@@ -6,7 +6,5 @@
   <link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">
   <link href="https://fonts.googleapis.com/css?family=Lato:700,900" rel="stylesheet">
   <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe" crossorigin="anonymous"></script>
-  {%- if jekyll.environment == 'production' and site.google_analytics -%}
-    {%- include google-analytics.html -%}
-  {%- endif -%}
+  {%- include analytics.html -%}
```

Comment:
Why do you need this change? Can't we use the pre-existing integration?

Reply:
I think that file is provided by the extension...

Reply:
To be honest, I am not sure. Should I revert the analytics then?

Reply:
I don't think changing the analytics belong together with a redesign, no...